src/assets/js/pages/ApplyPage/components/UserSection.js

```diff
@@ -127,7 +127,7 @@ class UserSection extends React.Component {
 
       {submitError && fields.createRow(
         fields.createColumn('col-sm-12',
-          fields.createError(submitError.message)
+          fields.createError(submitError.message + ' (Your Resume Upload Might Be Too Big)')
```

Comment:
Is there some way we can check for this, instead of just an ambiguous error?

Reply:
https://react-dropzone.js.org/#dropzone Lets try doing client-side, then? Check out maxSize